draft-scenarios/clash/arcane_artillery.tex

```diff
@@ -0,0 +1,155 @@
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles and arrange them as shown in the Scenario map layout:
+
+\textbf{For a 2-player Scenario:}
+\begin{itemize}
+  \item 2 × Starting (I) Map Tile
+  \item 4 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\textbf{For a 3-player Scenario:}
+\begin{itemize}
+  \item 3 × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\textbf{For a 4-player Scenario:}
+\begin{itemize}
+  \item 4 × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\textbf{For a 5-player Scenario:}
+\begin{itemize}
+  \item 5 × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\textbf{For a 6-player Scenario:}
+\begin{itemize}
+  \item 6 × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+To win the Scenario, a Hero must Flag the Dragon Utopia Field.
+
+\vspace*{\fill}
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+At the end of the \nth{12} Round, if there is no winner, all players lose the Scenario.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{6} Round:}
+\begin{itemize}
+  \item Remove all Black cubes from every Tile.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+Before the start of this Scenario:
+
+\begin{itemize}
+  \item If playing Alliance mode, allies and oppoents choose starting Tiles alternately.
+\end{itemize}
+
+During this Scenario:
+
+\begin{itemize}
+  \item Heroes cannot enter starting Tiles other than their own.
+  \item A player may not use the Diplomacy card to skip Combat on the Dragon Utopia Field.
+  \item Arcane Spikes are represented by cubes not used by any player's Faction.
+\end{itemize}
+
+\vspace*{\fill}
+
+\columnbreak
+
+\begin{itemize}
+    \item Obelisks may be captured. The first time an Obelisk is captured, gain one Arcane Spike.
+    \item At the start of each player's turn, that player removes any unspent Arcane Spikes, then gains Arcane Spikes equal to the number of captured Obelisks they have.
+\end{itemize}
+
+\vspace{1em}
+
+\includegraphics[width=\linewidth, keepaspectratio]{\art/griffin.png}
```

Comment:
You already used the griffin, please pick something else. Here's some inspiration: https://github.com/Heegu-sama/Homm3BG/tree/main/assets/art